sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -111,23 +111,25 @@ catch (Exception ex)
 }
 ```
 
-## Create and delete digital twins
+## Create, get, list, update, and delete digital twins and update components
```

Comment:
Did you mean to add both "get" and "list"?

Reply:
Yes, but now you mention it we don't do a list in this sample, so I'll remove it.